cmk/base/plugins/agent_based/utils/printer.py

```diff
@@ -37,6 +37,10 @@
     ".1.3.6.1.4.1.345",  # epson
     ".1.3.6.1.4.1.1248",  # seiko epson
     ".1.3.6.1.4.1.641.2",  # lexmark
+    ".1.3.6.1.4.1.641.52", # lexmark
+    ".1.3.6.1.4.1.641.1", # lexmark
+    ".1.3.6.1.4.1.641.3", # lexmark
+    ".1.3.6.1.4.1.641.51", # lexmark
```

Comment:
FYI: We use black as a formatter. You can install it on most linux distribution nowadays. You can simply run it on the whole cmk folder or the changed files and everything is up to standard.

Reply:
normally black is running automatically on my files i don't know why it used tabs instead of spaces here.

Reply:
my black was happy with this one space before the comment

Reply:
hm strange. Mine never was. But the CI is happy. So all good.